common/oatbox/extension/Manifest.php

```diff
@@ -197,7 +202,8 @@ public function getInstallPHPFiles(): array
     {
         $result = [];
         if (isset($this->manifest['install']['php'])) {
-            $result = is_array($this->manifest['install']['php']) ? $this->manifest['install']['php'] : [$this->manifest['install']['php']];
+            $result = is_array($this->manifest['install']['php'])
+                ? $this->manifest['install']['php'] : [$this->manifest['install']['php']];
```

Comment:
suggestion: Keep both branches in separate lines for consistency

```suggestion
            $result = is_array($this->manifest['install']['php'])
                ? $this->manifest['install']['php']
                : [$this->manifest['install']['php']];
```

Reply:
updated, thanks hector! :)